roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2

```diff
@@ -6,7 +6,7 @@ mcts.phone.number=8762963816
 js.directory.name=/ziggy
 form.directory.name=/form
 form.download.files=form.xml, model.xml, form_definition.json
-multimedia.directory.name={{ opensrp_openmrs_tomcat_user_home }}/{{ opensrp_openmrs_tomcat_instance }}/.OpenMRS/
+multimedia.directory.name={{ opensrp_openmrs_tomcat_user_home }}/.OpenMRS/
```

Comment:
@manutarus how about we use `openmrs_home_directory` instead of `opensrp_openmrs_tomcat_user_home`. So:

```
multimedia.directory.name={{ openmrs_home_directory }}
```